vagrant_config_default.json

```diff
@@ -19,12 +19,31 @@
   "provisioners": [{
     "_id": "0",
     "_ua_cookbooks_path": [
+      "workspace/dev-setup/chef/berks-cookbooks",
       "workspace/dev-setup/chef/main-cookbooks"
     ],
     "_ua_run_list": [
       "teracy"
     ],
     "json": {
+      // to work with kubernetes
+      "kubernetes-stack": {
```

Comment:
have you checked to make sure everything should work? I haven't seen the reciple (k8s.rb) to handle the  json config here yet. (check the similar setup on acme101/dev-setup

Reply:
fixed!